interface/src/app/gene/gene-network.tpl.html

```diff
@@ -3,7 +3,7 @@ <h1 class="page-header">
     Gene Network
   </h1>
   <div class="well">
-    <gene-search-form></gene-search-form>
+    <gene-search-panel organism="organism"></gene-search-panel>
```

Comment:
As far as I know, Angular only cares about whether the file specified by **templateUrl** exists and can be loaded, it doesn't really enforce any filename rules. The suffix "**.tpl.html**" is actually enforced by ngBoilerplace in **build.config.js** (lines 31-32). So I don't think we need to stick to a certain format, unless you guys want to.